src/browser/fetch_network.js

```diff
@@ -61,6 +61,11 @@ FetchNetworkAdapter.prototype.connect = function(port)
     return fake_tcp_connect(port, this);
 };
 
+FetchNetworkAdapter.prototype.tcp_probe = function(port)
```

Comment:
Out of curiosity, what do you use this for?

Reply:
Great question!  In v86 I have some tech worked out so that the service worker can receive web requests, and then route them from the browser into an httpd server running in the VM. I like the classic look of it just being a terminal, so it uses this to probe common ports in the background.  When it finds one open it adds a split pane with an iframe.  You don't want to actually initiate the connections, as that can often confuse servers, so similar to nmap, we dont send SYN ACK so the user-space program never sees the probing connections.

Reply:
Cool stuff, looking forward to playing around with that.